Dockerfiles/ocp4_content

```diff
@@ -8,10 +6,47 @@ RUN microdnf -y install cmake make git /usr/bin/python3 python3-pyyaml python3-j
 
 COPY . .
 
-RUN ./build_product --datastream-only --debug ocp4 rhcos4 eks
+# Enable the default.profiles as they maintain a list rules to be added to the datastream
+RUN find . -name "default\.profile" -exec sed -i 's/\(documentation_complete: \).*/\1true/' '{}' \;
```

Comment:
I don't think this is needed, since this conttainer file is not disabling all the profiles:
https://gitlab.cee.redhat.com/isc-devel/openshift-compliance-midstream/-/blob/rhaos-4.12-rhel-8/distgit/containers/openshift-compliance-content/Dockerfile.in?ref_type=heads#L18

@Vincent056 Maybe you intended all profiles to be disabled first? To follow same approach as downstream?

Reply:
I wanted to keep x86 as orignal as possible and only to change non-x86 to be same as downstream

Reply:
@Vincent056 Not sure what you mean with as close to original as possible? Do you mean as close to upstream as possible?